tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAsset.scala

```diff
@@ -21,18 +23,21 @@ case class PublicAssetId(value: UUID) {
 }
 
 object PublicURI {
-  def fromString(value: String): PublicURI = PublicURI(new URI(value))
-
-  def from(assetId: PublicAssetId, account: AccountId, publicURIPrefix: String = ""): PublicURI =
-    PublicURI.fromString(publicURIPrefix + s"/publicAsset/${account.id.value}/${assetId.asString()}")
+  def fromString(value: String): Either[Throwable, PublicURI] = Try(new URI(value))
+    .map(PublicURI.apply)
+    .toEither
+
+  def from(assetId: PublicAssetId, account: AccountId, publicUriPrefix: URI = new URI("")): PublicURI = {
```

Comment:
Does the default value `new URI("")` make sense?

IMO no

Maybe no default is better than bad default?